src/main/java/uk/gov/companieshouse/model/enums/OfficerRole.java

```diff
@@ -4,8 +4,14 @@
 
 public enum OfficerRole {
     DIRECTOR("director"),
+    CORPORATE_DIRECTOR("corporate-director"),
+    CORPORATE_NOMINEE_DIRECTOR("corporate-nominee-director"),
+    JUDICIAL_FACTOR("judicial-factor"),
     SECRETARY("secretary"),
```

Comment:
Is secretary a valid role?

Reply:
It is a role but it isn't valid for closing a company, so we don't actually use it outside of testing

Let me check why this is in the enum.